src/interpolation/detail/ArborX_InterpDetailsMovingLeastSquaresCoefficients.hpp

```diff
@@ -94,11 +94,11 @@ class MovingLeastSquaresCoefficientsKernel
     // - PHI is the diagonal weight matrix / CRBF evaluated at each source
     // point.
 
-    // We first change the origin of the evaluation to be at the target point.
-    // This lets us use p(0) which is [1 0 ... 0].
+    // We first change the origin of the evaluation to be at the target point
+    // within about unit distance. This lets us use p(0) which is [1 0 ... 0].
```

Comment:
I think it would be helpful to add a comment that we do this so that our vandermonde matrix is (what?)